src/app/shared/money-input/money-input.component.css

```diff
@@ -0,0 +1,32 @@
+input {
+  text-align: center;
+  background: #ffffff;
+  border: 1px solid #240b18;
+  box-sizing: border-box;
+  border-radius: 5px;
+  width: 164px;
+  height: 58px;
+  font-size: 24px;
+}
+
+input[type='number']::-webkit-inner-spin-button,
```

Comment:
FYI: using this you should be aware of https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-inner-spin-button